src/style.css

```
@@ -1,3 +1,70 @@
+:root {
+	--blue: #00acdc;
+}
+
 body {
-  margin: 0;
+	margin: 0;
+}
+
+.header {
+	display: flex;
+	align-items: center;
+	padding: 0 50px;
+	background: #fff;
+	box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+	justify-content: space-between;
+}
+
+.logo {
+	display: block;
+}
+
+.nav .nav__list {
+	display: flex;
+	align-items: center;
+	list-style: none;
+}
+
+.nav .nav__list .nav__item:not(:last-child) {
```

why increase specificity if this class in the code is unique?
```suggestion
.nav__item:not(:last-child) {
```

---

Sorry)
Forgot about this)